### ISSUE TITLE
negative predictions when fine-tuning

### ISSUE THREAD

**Issue by jinamshah** [google-research/bert#591]
Hi,
I am new to Tensorflow and BERT-like models, so please pardon me if this is a stupid question.
I tried to run this example after making changes to it according to my task: https://colab.research.google.com/github/google-research/bert/blob/master/predicting_movie_reviews_with_bert_on_tf_hub.ipynb

I get an accuracy of 75% on testing dataset.
I tried to run the estimator.prediction function on some data and got negative results. I thought this is because the model has not seen the data that I'm feeding and is given negative similarity for the said data. Hence, I tried to predict on one of the training data and got negative probabilities on this as well. Any ideas why?

**Comment by chikubee** [comment]
change in create_model() function can help. 
probs = tf.nn.softmax(logits, axis=-1)
instead of returning log_probs, return this.
